labelbox/data/annotation_types/__init__.py

```diff
@@ -32,6 +32,10 @@
 from .classification import Dropdown
 from .classification import Radio
 from .classification import Text
+from .classification import Prompt
+from .classification import ResponseText
```

Comment:
why are these new classes, i.e. why not reuse `Radio`, `Checklist` etc

Reply:
The reason why we decided to do it is because these are new feature schema kinds.

We can reuse the Text, Radio, and Checklist feature schema, which will work perfectly. We just wanted to separate them.

If you think it is unnecessary, then we can remove these new classes.

Reply:
My guess is you would add additional class or mappings for ndjson serialization / deserialization look here labelbox/data/serialization/ndjson/classification.py